browser/main/NoteList/index.js

```diff
@@ -259,13 +259,23 @@ class NoteList extends React.Component {
   }
 
   jumpNoteByHashHandler (event, noteHash) {
+    const { data } = this.props
+
     // first argument event isn't used.
     if (this.notes === null || this.notes.length === 0) {
       return
     }
 
     const selectedNoteKeys = [noteHash]
-    this.focusNote(selectedNoteKeys, noteHash, '/home')
+
+    let locationToSelect = '/home'
+    const notesByHash = data.noteMap.map((note) => note).filter((note) => note.key === noteHash)
```

Comment:
Why don't you use `find` instead of `filter`? Can you explain it to me?

Reply:
@ZeroX-DG Thanks for your Feedback. I was inspired by the `handleNoteClick` function in `NoteList/index.js`. I've changed the code and using the `find` function to get the first note and not all notes that matching the key (which always will be one).